games-emulation/rmg/metadata.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM 'https://www.gentoo.org/dtd/metadata.dtd'>
+<pkgmetadata>
+	<maintainer type="person" proxied="yes">
+		<email>orbea@riseup.net</email>
+		<name>orbea</name>
+	</maintainer>
+	<maintainer type="project" proxied="proxy">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<use>
+		<flag name="angrylion">Enable the mupen64plus-video-angrylion-plus XMAME licensed plugin</flag>
+		<flag name="asm">Enable the new dynarec for the mupen64plus-core plugin</flag>
```

Comment:
I wonder if this particular use flag could be a bit more descriptive? `dynarec`? (tbh I'd rename all plugin use flags to `x-plugin`, but that's just me)

Reply:
Both work for me, I changed them,

Reply:
You took my proposal a bit too literally :P I meant like:
 - angrylion-plugin
 - discord-plugin
 - dynarec-plugin
 - rust-plugin

Just so the use flag more accurately indicates to the user what it's about.

I can rename these while merging if you agree to it.

Reply:
Oops, thanks for clarifying. I fixed it.